src/ApiLoader/EnableInTraceeLinux.cpp

```diff
@@ -13,9 +13,9 @@
 #include <absl/hash/hash.h>
 #include <absl/strings/match.h>
 #include <dlfcn.h>
-#include <stdint.h>
 #include <sys/types.h>
 
+#include <cstdint>
```

Comment:
iwyu will not agree with clang-tidy here, because uint32_t is defined in stdint.h. cstdint only defines std::uint32_t

I would propose to move to std::uint32_t because we will need that for C++ modules anyway but there was a string opposition to that in the past (for good reason)